apps/dcos_net/test/dns-hostname.json

```diff
@@ -0,0 +1 @@
ey":"label","value":"MESOS_TASK_ID=test.f12cc459-cb38-11e8-a264-2e431fa5cff2"}],"force_pull_image":false}}}],"completed_tasks":[]},"get_executors":{"executors":[{"executor_info":{"executor_id":{"value":"test.f12cc459-cb38-11e8-a264-2e431fa5cff2"},"framework_id":{"value":"d0a07cf9-7424-426b-b138-2be9d07b8e62-0001"},"command":{"environment":{"variables":[{"name":"MARATHON_APP_VERSION","type":"VALUE","value":"2018-10-08T20:30:05.894Z"},{"name":"HOST","type":"VALUE","value":"localhost"},{"name":"MARATHON_APP_RESOURCE_CPUS","type":"VALUE","value":"0.1"},{"name":"MARATHON_APP_RESOURCE_GPUS","type":"VALUE","value":"0"},{"name":"MARATHON_APP_DOCKER_IMAGE","type":"VALUE","value":"library/ubuntu"},{"name":"MESOS_TASK_ID","type":"VALUE","value":"test.f12cc459-cb38-11e8-a264-2e431fa5cff2"},{"name":"MARATHON_APP_RESOURCE_MEM","type":"VALUE","value":"128.0"},{"name":"MARATHON_APP_RESOURCE_DISK","type":"VALUE","value":"0.0"},{"name":"MARATHON_APP_LABELS","type":"VALUE","value":""},{"name":"MARATHON_APP_ID","type":"VALUE","value":"/test"}]},"shell":false,"value":"/opt/mesosphere/packages/mesos--0ea3d8f2968796ab362056a15b84a374c7fa4409/libexec/mesos/mesos-executor","arguments":["mesos-executor","--launcher_dir=/opt/mesosphere/active/mesos/libexec/mesos"]},"container":{"type":"DOCKER","docker":{"image":"library/ubuntu","network":"HOST","privileged":false,"parameters":[{"key":"label","value":"MESOS_TASK_ID=test.f12cc459-cb38-11e8-a264-2e431fa5cff2"}],"force_pull_image":false}},"resources":[{"name":"cpus","type":"SCALAR","scalar":{"value":0.1},"allocation_info":{"role":"slave_public"}},{"name":"mem","type":"SCALAR","scalar":{"value":32},"allocation_info":{"role":"slave_public"}}],"name":"Command Executor (Task: test.f12cc459-cb38-11e8-a264-2e431fa5cff2) (Command: sh -c 'while true; ...')","source":"test.f12cc459-cb38-11e8-a264-2e431fa5cff2","discovery":{"visibility":"FRAMEWORK","name":"test","ports":{}}}}],"completed_executors":[]},"get_frameworks":{"frameworks":[{"framework_info":{"user":"root","name":"marathon","id":{"value":"d0a07cf9-7424-426b-b138-2be9d07b8e62-0001"},"failover_timeout":604800,"checkpoint":true,"role":"slave_public","hostname":"10.60.23.91","principal":"dcos_marathon","webui_url":"http://10.60.23.91:8080","capabilities":[{"type":"TASK_KILLING_STATE"},{"type":"GPU_RESOURCES"},{"type":"PARTITION_AWARE"},{"type":"REGION_AWARE"}]}}]},"get_agents":{"agents":[{"agent_info":{"hostname":"localhost","port":5051,"resources":[{"name":"ports","type":"RANGES","ranges":{"range":[{"begin":1025,"end":2180},{"begin":2182,"end":3887},{"begin":3889,"end":5049},{"begin":5052,"end":8079},{"begin":8082,"end":8180},{"begin":8182,"end":32000}]}},{"name":"disk","type":"SCALAR","scalar":{"value":13397.0}},{"name":"cpus","type":"SCALAR","scalar":{"value":4.0}},{"name":"mem","type":"SCALAR","scalar":{"value":14555.0}}],"id":{"value":"d0a07cf9-7424-426b-b138-2be9d07b8e62-S4"},"domain":{"fault_domain":{"region":{"name":"aws/us-east-1"},"zone":{"name":"aws/us-east-1d"}}}}}]}}}
```

Comment:
It would be nice to have the content of this file to be `jq .`ed.

Reply:
We had troubles with reviewing very long json test data and switch to one liners in DC/OS 1.12.